CharacterCreation/01_04_Species_Opaleites

```diff
@@ -0,0 +1,52 @@
+==============================
+Opalietes
+==============================
+
+===== Society =====
+
+The Opaleites are one of the discordant societies that the Terrans have come
+across so far. Moving from Trioch II to Trioch IV briefly united the clans 
+to save their species, but rapidly descended back into infighting. The clans 
+are heavily separated by subspecies, primarily informed by diet. However, many 
+of the subspecies within food groups do not agree either. 
+
+They are primarily a clan-based society. There is more emphasis on being a part
+of your 
+
+Types:
+*Biological
+*Scaled Ampibian
+*Varied Diets: Obligate Carnivores to Strict Herbivores
+*Cold-Blooded
```

Comment:
Do we want to specify whether they're bipedal?

Reply:
*Votes for specifying.

Also, "Ampibian"